Adds a unit test

asgrim opened the pull request.

asgrim committed:
Adds a unit test

choult commented on tests/IndexTest.php:
```diff
@@ -0,0 +1,19 @@
+<?php
+
+class IndexTest extends PHPUnit_Framework_TestCase
```
Missing license information

asgrim replied:
Do you have a license template you wish me to use?

choult commented on tests/bootstrap.php:
```diff
@@ -0,0 +1,11 @@
+<?php
+
+$command = 'php -S localhost:8000 -t ./web >/dev/null 2>&1 & echo $!';
```
composer.json requires a minimum PHP version of 5.3, but this feature was only introduced in 5.4

asgrim committed:
Update min PHP version for dev 5.4

For require-dev we need 5.4 as the test currently requires the PHP
built-in webserver. For live, it can operate fine on min 5.4, that is
the justification for the differing PHP versions.

choult commented:
```
/*
 * This file is part of the PHP Berkshire website project.
 *
 * (c) PHP Berkshire <info@phpberks.co.uk>
 *
 * For the full copyright and license information, please view the LICENSE
 * file that was distributed with this source code.
 */
```

asgrim committed:
Add license template to PHP files

asgrim commented:
All updated, thanks :)
